add documentation to migrate to Discovery mongodb (#5488)
ovh/docs: pages/public_cloud/public_cloud_databases/mongodb_11_howto_migrate_to_discovery/guide.en-gb.md

Several plans for MongoDB existed (Essential, Business, Enterprise) and recent update change it to Discovery, Production, Advanced.
This update will need some migration, to your cluster with a deprecated plan to a new one. In most of the case this migration are automatic.
But if you want to migrate to a new Discovery cluster from an deprecated Essential or Business cluster, you will need to do it manualy.

**This guide explains how to manually migrate data from an Essential or a Business cluster to a Discovery cluster for a Public Cloud Database MongoDB.**

## Requirements

- A [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
- A MongoDB database running on your OVHcloud Public Cloud Databases ([this guide](/pages/public_cloud/public_cloud_databases/databases_01_order_control_panel) can help you to meet this requirement)
- [Configure your MongoDB instance](/pages/public_cloud/public_cloud_databases/mongodb_02_manage_control_panel) to accept incoming connections

## Concept

> Public Cloud Databases MongoDB Discovery plan is for testing purpose. The migration from an deprecated plan cluster is not automatic.

Reviewer: ```suggestion
Public Cloud Databases MongoDB Discovery plan is intended for testing purposes.
The migration of a cluster using a deprecated plan is not automatic.
```

> However, you might want to do it anyway. The process to do it is to create a new Discoevry cluster then migrate data from your deprecated cluster with the CLI.

Reviewer: ```suggestion
If you want to perform such a migration (e.g. from an `essential` cluster to a `discovery` one), then the process is to first create a new Discovery cluster, then migrate data from your deprecated cluster with the CLI.
```
